libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -215,6 +219,7 @@ void SITL_State::_parse_command_line(int argc, char * const argv[])
         {"synthetic-clock", false,  0, 'S'},
         {"home",            true,   0, 'O'},
         {"model",           true,   0, 'M'},
+        {"config",          true,   0, 'c'},
```

Comment:
Out of curiosity, why do we introduce extra "config" while we can use "model" that is received as frame_str in all SITL constructors.

Reply:
The config is used to pass additional arguments to the scrimmage simulator such as visual and motion models. The model argument is only the frame such as "scrimmage-plane" or "scrimmage-copter" so it doesn't hold the additional parameters for scrimmage.